app/models/concerns/sufia/admin_set_behavior.rb

```diff
@@ -0,0 +1,64 @@
+module Sufia
+  module AdminSetBehavior
+    extend ActiveSupport::Concern
+
+    include Hydra::AccessControls::WithAccessRight
+    include CurationConcerns::Noid
+    include CurationConcerns::HumanReadableType
+    include CurationConcerns::HasRepresentative
+
+    included do
+      DEFAULT_ID = 'admin_set/default'.freeze
+
+      def self.default_set?(id)
+        id == DEFAULT_ID
+      end
+
+      validates_with HasOneTitleValidator
```

Comment:
This is duplicating stuff already on this class. https://github.com/projecthydra/curation_concerns/blob/master/app/models/concerns/curation_concerns/admin_set_behavior.rb#L11-L28  Can you keep the additions here just the ones that are different?